library/packages/test/slide_show_test.rb

```diff
@@ -150,6 +150,7 @@
 
   describe "#Redraw" do
     before do
+      textdomain "base"
```

Comment:
why it is needed? It should be not needed as there is not localized string ( and should not in test ).

Reply:
RuntimeError:
No textdomain configured in Yast::SlideShowClass, cannot translate "%s Release Notes"
in OSC. Up to now I have not found the right way.....

Reply:
this is for sure wrong, as it is already in main https://github.com/yast/yast-yast2/blob/master/library/packages/src/modules/SlideShow.rb#L123

Reply:
let me check it.

Reply:
can you point me to log where it failed? or can I reproduce it locally in your branch with rake osc:build?